### DISCUSSION TITLE
How to support for automatic connection setup between RDS and EC2, vice versa

### DISCUSSION THREAD

**Discussion by wowzoo** [aws/aws-cli#7355] [Q&A] [answered]
I read below two "what's new" articles

https://aws.amazon.com/about-aws/whats-new/2022/10/amazon-ec2-automated-connection-solution-ec2-instance-rds-database/?nc1=h_ls

https://aws.amazon.com/about-aws/whats-new/2022/10/amazon-rds-automating-connectivity-set-up-existing-rds-aurora-database-ec2-compute-instance/?sc_channel=em&sc_campaign=weekly_whats_new_korea&sc_medium=em_whats_new&sc_content=launch_ot_ot&sc_country=mult&sc_geo=mult&sc_category=mult&sc_outcome=launch

then looked up AWS CLI command list for EC2 and RDS. but I can't find command for those features. 
I wonder if those features only work on AWS Console, if it does, then is there any plan to put them in AWS CLI?

**Comment by RyanFitzSimmonsAK** [comment, marked as answer]
Hi @wowzoo, thanks for reaching out.  You're correct that this feature is not currently usable through CLI; new features sometimes are available only through console at first.  I've reached out to the team to figure out if API support is being currently worked on, and I'll let you know when I hear back.  Let me know if you have any further questions.
